### ISSUE TITLE
Automation Rule-Model Editor destroys String in rule script

### ISSUE THREAD

**Issue by njrl** [cdjackson/HABmin#213]
when i use special sign in my script like " °C " the editor stores the script in the right way (checked in a external editor). But when I reopen the next time the degree sign will be destroed.
before:
rule "TemperaturString"
when
	System started
	or Item Temperature_R2_Aussen changed
	or Item Temperature_R2_Innen changed
then
	var String msg = ""
	msg = Temperature_R2_Aussen.state.format("%.1f") + " ��C / " + Temperature_R2_Innen.state.format("%.1f" + " ��C")
	postUpdate(Weather_ShortString,msg)
end

**Comment by cdjackson** [comment]
Can you confirm if the file is saved correctly? I want to see if the problem is with saving, or reading back.=

**Comment by njrl** [comment]
that was what i try to say... when i change it in the inline editor he stores it correctly. until i open it again ...  it will be destroyed in the moment of  import  in the editor .... 
that there are two wrong signs instread of one wanted, tells me that this is a utf-8 converting problem(?)

**Comment by cdjackson** [comment]
Yes - it looks like a UTF-8 issue which is strange since I fixed that a long time ago (see #121)…  I’ll take a look…

**Comment by njrl** [comment]
ok .. say if i can help further.
I'm on an ubuntu with latest firefox and chrome 
OH runs on an raspberry with java version "1.8.0-ea"
